### ISSUE TITLE
some rule file tests failing

### ISSUE THREAD

**Issue by d-w-moore** [irods/irods#7794]
- [ ] 4-2-stable
   - [ ] main
   ---
 
Fix failing test(s) in icommands rule file test suite.

The following are related changes that allow these tests to run (spec. on `main`) and accurately reflect the results of successful / failed rule runs as test passes and fails respectively.
 
https://github.com/irods/irods/pull/6206
https://github.com/irods/irods/pull/6170

**Comment by alanking** [comment]
@d-w-moore - Do you know whether this is still true? Maybe we are skipping these tests at the moment.

**Comment by alanking** [comment]
I assume this refers to `test_all_rules.Test_AllRules` and the microservices found here: https://github.com/irods/irods_client_icommands/tree/0cb6cd700e2111fdc1ab214daedf85e1a4da9055/test/rules. The linked PRs have been merged and the associated issue https://github.com/irods/irods/issues/6169 has been resolved.

This issue does not refer to any specific tests and our test suite has been passing (and when things have failed, we have created issues). I think this can be closed as resolved/invalid. Should we encounter failing tests, we can open new issues for those.
